tools/vp20compiler/main.c

```diff
@@ -520,7 +519,7 @@ void translate(const char* str)
                 } else if (reg.File == PROGRAM_ENV_PARAM) {
                     vsh_set_field(vsh_ins, mux_field[j], PARAM_C);
                     // TODO: the index needs ajustment?
-                    vsh_set_field(vsh_ins, FLD_CONST, reg.Index+96);
+                    vsh_set_field(vsh_ins, FLD_CONST, reg.Index);
```

Comment:
Drop this commit for now and create an issue about this - consider how this could / should work and how the encoding is.